### ISSUE TITLE
website is non-compliant with accessibility laws

### ISSUE THREAD

**Issue by TVW96** [TVW96/github-issue-template#3]
**Describe the bug**
I checked the site with an accessibility checker and the explore California site failed to pass inspection.

**To Reproduce**
1. Go to 'https://www.accessibilitychecker.org/'
2. Paste in 'https://tvw96.github.io/github-issue-template/'
3. See error status: NON-COMPLIANT 

**Expected behavior**
The website should pass accessibility checker inspection.

**Desktop (please complete the following information):**
 - n/a

**Smartphone (please complete the following information):**
 -n/a

**Additional context**
Website suggest using audioeye to help with making our website accessible and compliant under United State law.
